Assignment/Book-Management/src/app/core/services/book.service.ts

```diff
@@ -27,9 +35,87 @@ export class BookService {
         this.totalItems = objTemp.totalItems;
       })).subscribe()
 
-    return this.http.get<any>(environment.apiLink+`/books?pageNumber=${pageIndex}&pageSize=${pageSize}`, {
+    return this.http.get<any>(environment.apiLink+`/books?pageNumber=${pageIndex}&pageSize=${pageSize}`)
+  }
+
+  getBook(id: number) {
+    return this.http.get<any>(environment.apiLink+`/books/${id}`, {
+      headers: new HttpHeaders({
+        Authorization: 'Bearer ' + environment.token
+      })
+    })
+  }
+
+  getAuthor(id: number) {
+      this.author = {};
+      this.authorService.getAuthorById(id).subscribe(val => {
+        this.author = {
+          "id": val.id,
+          "name": val.name,
+          "website": val.website,
+          "birthday": `${val.birthday}`,
+          "cover": null
+        }
+      });
```

Comment:
Already talked about this. Don't use "string quote" if not necessary.

Reply:
Done. Because I haven't merged yet, so it not update